scrapy/settings/default_settings.py

```diff
@@ -158,6 +159,7 @@
 }
 FEED_EXPORT_INDENT = 0
 
+FEED_STORAGE_GCS_ACL = None
```

Comment:
I still wonder if that's ok to use None values as defaults, instead of empty strings, because there is no way to set an option to None using `-s` from command line (or am I wrong here?). So, once overridden, you can't get it back to the default value, when starting a spider. That's the reason most other default values are empty strings instead of None - we should probably treat empty string as None in most cases. By having it `''` in default_settings, and checking that it works, we make sure that `-s FEED_STORAGE_GCS_ACL=` from command line also works.